active_learning_multiannotator/active_learning_single_annotator.ipynb

```diff
@@ -0,0 +1,461 @@
+{
+ "cells": [
+  {
```

Comment:
Move this entire notebook and helper file to a separate folder. Add that folder to table of contents with description of this example (list it after the multi-annotator AL example).  Make sure description is concise but makes it clear how this is different than the active learning multi-annotator notebook.